### ISSUE TITLE
8bit Isaac - Isaac Newton Game (prototype) 

### ISSUE THREAD

**Issue by instinct** [instinct/GameFroot#68]
One of my own personal goals with the game creator is to create a range of serious games that teach people about significant figures throughout history. 

One would be Isaac Newton and another might be say Shakespeare. 

Initially I think we should create an 8bit styled "young Newton" character (possibly like a young Braid character) that would be the main game hero. Both young Newton and the main level would be in the following 8 bit palette:

Initial palette: http://8bitflashgames.com/escape-from-the-planet-of-robot-monsters/

I'd like to see collectable items splattered around in a different 8bit color palette though. This would make them stand out nicely during moments of significance in the game.

The game is a platform game set in the old days when Newton would have been young. 

Significant things about Newton;

1) Principia Mathmatica (the basic principals of behind a lot of the maths that we still use today - ie GPS
2) Discovered Gravity
3) Did a research on Optics and Color
4) Was alive for the Black Plague and survived

So the level could be set in a plague set England. There should be sick looking Rats that you have to jump over. And there should be sick looking people that you must avoid. 

Game would be about not getting sick and collecting objects within a set time that educate the player about Newtons discoveries - but through the Game Play not obvious story telling.

Maybe one level you have to collect something that triggers something else that causes an apple to fall out of a tree and onto young Newtons head. Boom!

Another level could be about collecting enough coins to buy a prism, the prism that Newton used in his studies to explain colors and light. 

This could be fun.

**Comment by instinct** [comment]
Idea:  puzzles or challenges for each stage relating to the particular achievements of Newton - ie, not just 'collecting' an apple, but to complete a stage you need to complete tasks that relate to gravity and the player understanding the principles of it?  
Collect prism, that is then needed in the Colour stage to work out his discoveries with optics

**Comment by thornyd** [comment]
We have had lots of good ideas about this, now we need to make a solid plan with direction for the game.

**Comment by instinct** [comment]
Yes I agree. We need to add some solid ideas to the game concept and sketch them up. Makes me think that we spend a little more time converting the first sketch into an 8bit (maybe 16bit) palette and then we just do more sketching...

So far we've got what might be a good starting point for the game. He's just come out of that door. Then it goes side scrolling and our character heads off to the right. You've got the nice arches. Some towers. Some lightening. Nice atmosphere. I think we need to work out what else is along the stage / what the user walks by. Perhaps a couple of shops. A stall. A few things that the person can walk by and interact with. Nothing to kill / hurt you yet.

Things to sketch....
- Cemetery level. Base it off Ghosts and Goblins level design. This level would be about alchemy like thornyd suggests. Also with the addition of bodies with blankets over them - something that illustrates people that have died of the plague.  
- Garden level. A garden level... with a tree.
- London Burning Level. Lots of rats to jump over. This is the Plague level. 
- Catacombs level. Maybe kind of like the underground mission in the Goonies. 
- Isaac Newton's study and childhood study?? Something academic. A puzzle?

What about you Dave. Any other ideas?

**Comment by marsh0lion** [comment]
This is being acted upon and hasn't had any action in a long time
